stdlib/public/CTensorFlow/ctensorflow_init.cpp

```diff
@@ -43,6 +43,46 @@ void *swift_tfc_CreateScalarFloatTensor(int32_t val) {
   return tensor;
 }
 
+void *swift_tfc_CreateScalarIntTensor(int64_t val, int32_t dtype,
```

Comment:
I'm not sure why ints and floats have to be in different code paths. Can you explain a bit more, and also add some doc comments at the declaration site?

Reply:
The float version should take as input a float or double value, instead of `int64_t val`, so there should be different C APIs for them.

BTW, currently we only have integer related data types like IGM.Int32Ty, but there is no IGM.FloatTy. That can be added though.

Reply:
Done.